tests/test_linestring.py

```diff
@@ -132,16 +132,6 @@ def test_linestring(self):
                          {'type': 'LineString',
                           'coordinates': ((1.0, 2.0), (3.0, 4.0))})
 
-    @shapely20_deprecated
-    def test_linestring_mutate(self):
```

Comment:
Perhaps rename this `test_linestring_immutable` and check that `.coord` raise TypeError.